abel/basex.py

```diff
@@ -259,54 +273,70 @@ def get_bs_basex_cached(n_vert, n_horz,
         the matrices that compose the basis set.
     """
 
+    global _prm, _M, _reg, _LR
```

Comment:
Yikes, is making these global variables the only way to cache these things?

Reply:
They are local to this module. From user's perspective, they are in principle accessible as `abel.basex._prm` and so on, but otherwise have no effect.

Reply:
Okay, that seems reasonable.